builtin/logical/pki/path_fetch_issuers.go

```diff
@@ -0,0 +1,258 @@
+package pki
+
+import (
+	"context"
+	"encoding/pem"
+	"regexp"
+	"strings"
+
+	"github.com/hashicorp/vault/sdk/framework"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+var nameMatcher = regexp.MustCompile("^" + framework.GenericNameRegex("ref") + "$")
+
+func pathListIssuers(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "issuers/?$",
+
+		Callbacks: map[logical.Operation]framework.OperationFunc{
+			logical.ListOperation: b.pathListIssuers,
+		},
+
+		HelpSynopsis:    pathListIssuersHelpSyn,
+		HelpDescription: pathListIssuersHelpDesc,
+	}
+}
+
+func (b *backend) pathListIssuers(ctx context.Context, req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
```

Comment:
Could we rename to pathListIssuersHandler, using the same function name only separated by a func vs receiver is triggering me ;)